gapic/templates/%namespace/%name_%version/%sub/services/%service/async_client.py.j2

```diff
@@ -243,6 +251,9 @@ class {{ service.async_client_name }}:
 
         )
 
+        if CLIENT_LOGGING_SUPPORTED:
+            _LOGGER.debug(" {{ service.async_client_name }} client is successfully constructed.")
```

Comment:
The `ruby` logs says `Created client for google.cloud.secretmanager.v1.SecretManagerService` which matches the proto: https://github.com/googleapis/googleapis/blob/072ae46e493035d1419caad40bccc1a4f1b977bf/google/cloud/secretmanager/v1/service.proto#L46

The equivalent log for showcase would be `Created client for google.showcase.v1beta1.Echo` instead of what we currently have `EchoClient client is successfully constructed.` 
https://github.com/googleapis/gapic-showcase/blob/625d292c5574c703f42bf50f7ea7b92d47f4918e/schema/google/showcase/v1beta1/echo.proto#L42

I would prefer `Created client for google.showcase.v1beta1.Echo`. WDYT?